Bump zenko 2.6.17

francoisferrand opened the pull request:
- Bump cloudserver 8.7.15
- Bump version 2.6.17

Issue: ZENKO-4555

francoisferrand committed:
Bump cloudserver 8.7.16

Issue: ZENKO-4555

francoisferrand committed:
Bump version 2.6.17

Issue: ZENKO-4555

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko/doc/user).





*Status report is not available.*

francoisferrand committed:
Fix mongo-kafka-connect download url

Issue: ZENKO-4555

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/2.7/improvement/ZENKO-4555` with contents from `improvement/ZENKO-4555`
and `development/2.7`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/2.7/improvement/ZENKO-4555 origin/development/2.7
 $ git merge origin/improvement/ZENKO-4555
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/2.7/improvement/ZENKO-4555
```

ghost commented:
FYI: currently blocked by https://github.com/scality/Arsenal/pull/2115 cc @francoisferrand

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

francoisferrand commented:
/approve

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/e70ca203fce805ff2844ca1234da8127a65c564e) did not succeed in branch improvement/ZENKO-4555.




*The following options are set:* **approve**

francoisferrand committed:
Upgrade to kind 1.12 (and k8s 1.23.4) in CI

This includes a fix for [1], so ulimit is not stuck to 1024.

[1] https://github.com/kubernetes-sigs/kind/pull/2565

Issue: ZENKO-4555

francoisferrand committed:
Disable failing "delete" notifcations

Issue: ZENKO-4555

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/bae9b6c234b89b8ce2323ed119dd8675d57d06ea) did not succeed in branch w/2.7/improvement/ZENKO-4555.




*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.6`

* :heavy_check_mark: `development/2.7`



The following branches will **NOT** be impacted:

* `development/2.4`
* `development/2.5`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.6`

* :heavy_check_mark: `development/2.7`



The following branches have **NOT** changed:

* `development/2.4`
* `development/2.5`



Please check the status of the associated issue ZENKO-4555.

Goodbye francoisferrand.
